### DISCUSSION TITLE
Class `stroke-current` ignored during build

### DISCUSSION THREAD

**Discussion by feymo** [tailwindlabs/tailwindcss#10934] [Help] [answered]
### Local information
- TailwindCSS --version: v3.2.7
- Node --version: v19.8.1
- Browser: N/A
- OS: macOS
---

Following [documentation](https://tailwindcss.com/docs/stroke), i tried to apply the `stroke-current` class on a `svg` element : 

```html
<svg xmlns="http://www.w3.org/2000/svg" class="stroke-current" viewBox="0 0 24 24">...</svg>
```
However, while `build` (dev or production mode) and `watch`, the `stroke-current` class seems to be ignored as no related rule appears in my final asset file.
But if i add a custom theme definition in my `tailwind.config.js` file such as : 
```javascript
module.exports = {
    theme: {
        extend: {
            stroke:{
              current: 'currentColor',
            },
        },
    },
}
```

then the class appears and everything is ok.
As i understood from the documentation, the `stroke-current` class seems to be a native one from tailwindcss, 🤔  am i wrong ?

Thanks for your feedbacks ! 👍🏼

**Comment by wongjn** [comment, marked as answer]
Perhaps check whether you have completely overridden the color palette in Tailwind and thus unsetting `current` as a color value and thus unsetting `stroke-current` as a valid class candidate.

**Reply by feymo** [reply]
That's exactly the case!
I had no idea that "current" was defined as a color in the tailwind palette but it makes sense 👍🏼
